core-services/ipi-deprovision/aws.sh

```
@@ -268,49 +270,156 @@ function vpc_has_only_orphaned_eni() {
 # log installer version for debugging purposes
 openshift-install version
 
-clusters=$( find "${logdir}" -mindepth 1 -type d )
-for workdir in $(shuf <<< ${clusters}); do
-	queue deprovision "${workdir}"
-done
-
-wait
+clusters=$( find "${logdir}" -mindepth 1 -type d 2>/dev/null || true )
+if [[ -n "${clusters}" ]]; then
+	for workdir in $(shuf <<< ${clusters}); do
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)